tests/check/check_h26xsigned_auth.c

```diff
@@ -1908,16 +1906,18 @@ START_TEST(public_key_on_validation_side_later)
   sv_rc = signed_video_add_nalu_and_authenticate(sv_vms, i_nalu->data, i_nalu->data_size, &auth_report);
   ck_assert_int_eq(sv_rc, SV_OK);
   ck_assert(!auth_report);
-  sv_rc = signed_video_add_nalu_and_authenticate(sv_vms, sei->data, sei->data_size, &auth_report);
-  ck_assert_int_eq(sv_rc, SV_NOT_SUPPORTED);
   // Set public key
-  sv_rc = signed_video_set_public_key(sv_vms, algo, sv_camera->signature_info->public_key, sv_camera->signature_info->public_key_size);
+  sv_rc = signed_video_set_public_key(sv_vms, sv_camera->signature_info->public_key, sv_camera->signature_info->public_key_size);
```

Comment:
Continue validation after this call. That should work.
Also, there is no need for `i_nalu_2` and `sei_2`. Remove them.